### ISSUE TITLE
List of images being saved incorrectly

### ISSUE THREAD

**Issue by carlmjohnson** [decaporg/decap-cms#6918]
I upgraded from netlify-cms 2 to decap-cms 3. I have a line in my config to create lists of images:

```yml
      - label: "Headshot"
        name: "images"
        widget: list
        summary: "{{fields.image}}"
        field: { label: Image, name: image, widget: image }
```

In decap 3 but not netlify-cms 2, this saves incorrectly. I want it to produce `"image": [ "/file.jpeg" ]` in the content file (which is JSON in case that matters) but instead it produces `"image": [ [ "/file.jpeg" ] ]`.

**Comment by demshy** [comment]
Hi @carlmjohnson, sorry it took a while, been busy elsewhere. This should be fixed in latest version (3.0.8), do you mind trying and confirming?
Thanks

**Comment by carlmjohnson** [comment]
Yes, it appears to work now.
